src/pages/Checkout/CheckoutPage.vue

```diff
@@ -1,18 +1,25 @@
 <template>
 	<www-page>
-		<kv-page-container id="checkout-slim" data-testid="checkout" class="tw-py-3.5">
+		<kv-page-container
+			id="checkout-slim"
+			data-testid="checkout"
+			class="tw-py-3.5"
+			:class="{
+				'not-logged-in': !isLoggedIn,
+				'login-active': isLoggedIn,
+				'login-guest': checkingOutAsGuest
+			}"
```

Comment:
New set of classes to help determine checkout stage and state during automated testing.